ServerCore/Pages/FilesController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using ServerCore.DataModel;
+using ServerCore.Models;
+
+namespace ServerCore.Pages
+{
+    /// <summary>
+    /// Controller to redirect users to files in blob storage
+    /// </summary>
+    public class FilesController : Controller
+    {
+        private readonly PuzzleServerContext _context;
+
+        public FilesController(PuzzleServerContext context)
+        {
+            _context = context;
+        }
+
+        [Route("{eventID}/Files/{filename}")]
```

Comment:
Rightly or wrongly, we've been using eventId, not eventID. I am fine with either but we should standardize.

Reply:
I'll switch to eventId